src/util/options.schema.json

```
@@ -1407,6 +1400,20 @@
             "Path to C preprocessor (cpp) to use. If empty, then automatically searched.",
           "type": "string",
           "default": ""
+        },
+        "arrays-domain": {
+          "title": "exp.arrays-domain",
+          "description":
+            "The domain that should be used for arrays. trivial/partitioned/unroll. When employing the partition array domain, make sure to enable the expRelation analysis as well. When employing the unrolling array domain, make sure to set the exp.array-unrolling-factor >0.",
+          "type": "string",
```

Since there are only fixed string values acceptable for this option, they should be specified by the schema:
```suggestion
          "type": "string",
          "enum": ["trivial", "partitioned", "unroll"],
```

---

Fixed on d7fed6da2acc18c9211cce3cd5db38a47d37e5f5